docs/source/clusters-feds/backup.md

```diff
@@ -102,3 +113,11 @@ Although it's not advertised as such, RethinkDB's built-in replication feature i
 * Send a RethinkDB reconfigure command to the RethinkDB cluster to make it so that the `original` set has the same number of replicas as before (or maybe one less), and the `backup` set has one replica. Also, make sure the `primary_replica_tag='original'` so that all primary shards live on the `original` nodes.
 
 The [RethinkDB documentation on sharding and replication](https://www.rethinkdb.com/docs/sharding-and-replication/) has the details of how to set server tags and do RethinkDB reconfiguration.
+
+Once you've set up a set of backup-only RethinkDB servers, you could make a point-in-time snapshot of their storage devices, as a form of backup.
+
+You might want to disconnect the `backup` set from the `original` set first, and then wait for reads and writes in the `backup` set to stop. That way, all the data in the `backup` set will be consistent before you take the snapshot.
```

Comment:
It seems to me that the `backup` set would have just one copy of each shard (i.e. but no primary replicas). I don't see how there could be inconsistency between the servers of the `backup` set.

The servers in the `backup` set could be made inaccessible from clients (i.e. they only communicate with other RethinkDB servers, never with the outside world). If you disconnect the `backup` set from the `original` set, it becomes isolated, with no reads or writes coming from outside. It becomes a static island of data, so you can take a snapshot.

I should add these ideas to the text, assuming I didn't make a mistake in my reasoning. Did I?